tests/longevity/longevity_test.go

```diff
@@ -513,6 +518,28 @@ func populateIntervals() {
 	triggerInterval[VolumeResize][2] = 24 * baseInterval
 	triggerInterval[VolumeResize][1] = 27 * baseInterval
 
+	triggerInterval[PoolResizeDisk][10] = 1 * baseInterval
+	triggerInterval[PoolResizeDisk][9] = 3 * baseInterval
+	triggerInterval[PoolResizeDisk][8] = 6 * baseInterval
+	triggerInterval[PoolResizeDisk][7] = 9 * baseInterval
+	triggerInterval[PoolResizeDisk][6] = 12 * baseInterval
+	triggerInterval[PoolResizeDisk][5] = 15 * baseInterval
+	triggerInterval[PoolResizeDisk][4] = 18 * baseInterval
+	triggerInterval[PoolResizeDisk][3] = 21 * baseInterval
+	triggerInterval[PoolResizeDisk][2] = 24 * baseInterval
+	triggerInterval[PoolResizeDisk][1] = 27 * baseInterval
+
+	triggerInterval[PoolAddDisk][10] = 1 * baseInterval
+	triggerInterval[PoolAddDisk][9] = 3 * baseInterval
+	triggerInterval[PoolAddDisk][8] = 6 * baseInterval
+	triggerInterval[PoolAddDisk][7] = 9 * baseInterval
+	triggerInterval[PoolAddDisk][6] = 12 * baseInterval
+	triggerInterval[PoolAddDisk][5] = 15 * baseInterval
+	triggerInterval[PoolAddDisk][4] = 18 * baseInterval
+	triggerInterval[PoolAddDisk][3] = 21 * baseInterval
+	triggerInterval[PoolAddDisk][2] = 24 * baseInterval
+	triggerInterval[PoolAddDisk][1] = 27 * baseInterval
```

Comment:
are we going to resize pool by adding disk every 3 minutes? As far as I know resizing pools take more than 3 minutes depending of the pool size

Reply:
no the minimum frequency will be atlease a day